planning/behavior_path_planner/behavior_path_planner_path_generation.md

```diff
@@ -0,0 +1,91 @@
+## Path Generation
+
+This document explains how the path is ganerated for lane change and avoidance, etc.
+
+### Overview
+
+The base idea of the path generation in lane change and avoidance is to smoothly shift the reference path, such as the center line, in the lateral direction. This is achieved by using a constant-jerk profile as in the figure below. More details on how it is used can be found in [README](./README). It is assumed that the reference path is smooth enough for this algorithm.
```

Comment:
> More details on how it is used can be found in [README](./README)
This seems to be pointing to non-existent page.

Reply:
Thank you! Done in https://github.com/autowarefoundation/autoware.universe/pull/2428/commits/067612f4ab6a9ad59b2461340af9238d55d468f2